docs/index.md

```diff
@@ -65,20 +67,20 @@ POST {json file} <url>/insert/
   "race": "2054-5", 
   "ethnicity": "2186-5",
   "visit": [
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"},
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}, 
-    {"date": "2017-07-25 15:02:54.171755", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"}
+    {"date": "2017-07-25 15:02:54.171-0:00", "visit_id": 1234567891011121, "location": "333 S State St", "provider": "John Doe"},
```

Comment:
The documentation spec says three decimal points for the timestamp. This has two.

Reply:
I think the decimals are there (that's the .171 part) but the 0:00 should be 00:00.